arm-compute/quickstart-templates/aliases.json

```diff
@@ -101,16 +143,9 @@
             "sku": "2012-Datacenter",
             "version": "latest",
             "architecture": "x64"
-          },
-          "Win2008R2SP1": {
-            "publisher": "MicrosoftWindowsServer",
-            "offer": "WindowsServer",
-            "sku": "2008-R2-SP1",
-            "version": "latest",
-            "architecture": "x64"
```

Comment:
I'd like to confirm whether directly removing this alias will not cause a breaking change to the customer? Is this image SKU `2008-R2-SP1` already invalid, right?

Reply:
@zhoxing-ms , it was removed from PS as the image is deprecated. It should already be nonfunctional. I tried using the alias with this command and received an error:
adam@Azure:~$ az vm create -n vmnam -g adsandimg --image Win2008R2SP1
Can't resolve the version of 'MicrosoftWindowsServer:WindowsServer:2008-R2-SP1'

Reply:
Feel free to reach out to PMs about this, I'm not exactly sure who to ask about this as my last contact has left Microsoft.

Reply:
Confirmed with Vishal Bajaj (Windows PM), this image should be retired.